MdeModulePkg/Include/Guid/CacheAttributesChangeEvent.h

```diff
@@ -0,0 +1,16 @@
+/** @file
+  Event group triggered when caching attributes are updated in the system.

```

Comment:
Can be nice to add a MU_CHANGE: Whole File, for this and other new files

Reply:
It also probably would be nice to split this into two commits and do a rebase and FF PR: first commit is the bulk of it, just the splitting off of MpDxe. Second commit is small and just the things we had to add (should just be this event and whatever you had to add).